packages/web-frontend/src/components/View/MatrixWrapper/MatrixWrapper.js

```diff
@@ -222,23 +222,33 @@ export class MatrixWrapper extends Component {
   };
 
   renderPeriodSelector() {
-    const { viewContent } = this.props;
+    const { viewContent, urlStartDate, urlEndDate } = this.props;
     const { periodGranularity } = viewContent;
     const { isLoading } = this.state;
 
     if (!periodGranularity) {
       return null; // Not using a period selector
     }
 
-    const datePickerLimits = getLimits(viewContent.periodGranularity, viewContent.datePickerLimits);
+    const { startDate: defaultStartDate, endDate: defaultEndDate } = getDefaultDatePickerDates(
+      viewContent,
+    );
+
+    const startDate = urlStartDate || defaultStartDate;
+    const endDate = urlEndDate || defaultEndDate;
```

Comment:
The URL reigns supreme! No matter what we respect the url dates, but in their absence we must show the default date.

This logic is duplicated in a couple places, so will refactor to a common util

Reply:
Did you want to come back to this?

Reply:
Ah thanks for the reminder! I forgot 🤦  will do 👍